docs/ref/contrib/admin/index.txt

```diff
@@ -2101,6 +2101,26 @@ templates used by the :class:`ModelAdmin` views:
         def get_changeform_initial_data(self, request):
             return {'name': 'custom_initial_value'}
 
+.. method:: ModelAdmin.get_formset_kwargs(request, obj, inline, prefix)
+
+    .. versionadded:: 4.0
+
+    A hook for formset parameters or kwargs.This hook can be used to provide
+    customize formset parameters. To use this hook, overwrite the function for
+    the respective Admin class in ``admin.py``. The overridden function should
+    return a dictionary which will consist of 2 parts, the first one will call
+    the parental function which will return the key-value pairs of default
+    parameters and the other one should be a key-value pair in which the key
+    would be ``form_kwargs`` and the value would be a dictionary of customized
+    parameters. For example, if there is a need to customize the initial value
+    of formset then the following code may be used::
```

Comment:
I think this is rather wordy. It is a bit repetitive and gets into irrelevant details about `form_kwargs`. I think the example below is sufficient to show of the use of `form_kwargs`.

(Sorry, I've run out of time to review the rest of this paragraph...)